File: Day 10/Data Structures and Algorithms/linearSearch.py
def linear_search(array, num):
	for i in range(len(array)):
		if (array[i]==num):
			return i
	return -1

print(linear_search([1, 2, 3, 4, 5, 6, 7, 8, 9, 10], 8))

File: Day 10/Data Structures and Algorithms/linearSearch.js
function linearSearch(arr, item) {
    // Go through all the elements of arr to look for item.
    for (var i = 0; i < arr.length; i++) {
        if (arr[i] === item) {
            // Found it!
            return i;
        }
    }
    // Item not found in the array.
    return null
}

function verify(index) {
    if (index != null) {
        console.log("Target found at index : ", index )
    } else {
        console.log("Target not found")
    }
}

let result = linearSearch([1, 2, 3, 4, 5, 6, 7, 8, 9, 10], 8);
verify(result)
